### DISCUSSION TITLE
How to load full quality image?

### DISCUSSION THREAD

**Discussion by PhanVanLinh** [coil-kt/coil#1044] [Q&A] [answered]
I have some image that use in many screen (on each screen each will have different size). For example, it can display as a thumbnail (in mini player) then it can expand to fullscreen after click. 
Because, I load this image into mini player first then after the image expand to fullscreen, it become blurry. 
I try `precision(Precision.EXACT)`, `scale(Scale.FILL)` but it's not working. 
I migrate from fresco, how can I load full quality image?

**Comment by colinrtwhite** [comment, marked as answer]
`size(OriginalSize)` or `size(Size.ORIGINAL)` in 2.x
